curp/tests/common/curp_group.rs

```diff
@@ -117,8 +117,19 @@ impl CurpGroup {
                 let (as_tx, as_rx) = mpsc::unbounded_channel();
                 let ce = TestCE::new(name.clone(), exe_tx, as_tx, xline_storage_config);
 
-                let cluster_info = Arc::new(ClusterInfo::new(all_members.clone(), &name));
-                all = cluster_info.all_members_addrs();
+                let cluster_info = Arc::new(ClusterInfo::new(
+                    all_members
+                        .clone()
+                        .into_iter()
+                        .map(|(k, v)| (k, vec![v]))
+                        .collect(),
+                    &name,
+                ));
+                all = cluster_info
+                    .all_members_addrs()
+                    .into_iter()
+                    .map(|(k, mut v)| (k, v.pop().unwrap()))
+                    .collect();
```

Comment:
Please change `all` and `all_members` to multiple address

Reply:
The `Rpc::run_from_listener` startup mode can only accept one tokio TcpListener, so only one address can be used. I don't think we need to worry about multiple addresses during testing.